extendeddatacrossword_test.go

```diff
@@ -56,10 +56,15 @@ func TestRepairExtendedDataSquare(t *testing.T) {
 		flattened[4], flattened[5], flattened[6], flattened[7] = nil, nil, nil, nil
 		flattened[8], flattened[9], flattened[10] = nil, nil, nil
 		flattened[12], flattened[13], flattened[14] = nil, nil, nil
-		_, err = RepairExtendedDataSquare(original.getRowRoots(), original.getColRoots(), flattened, codec, NewDefaultTree)
+
+		var extHeader *ExtendedDataSquare
+		extHeader, err = RepairExtendedDataSquare(original.getRowRoots(), original.getColRoots(), flattened, codec, NewDefaultTree)
```

Comment:
nit: this is not really a header, right? maybe just call the var repairedEDS, or, simply eds?

Reply:
Renamed to `eds`